src/avatar.scss

```diff
@@ -97,17 +97,21 @@ $block: #{$fd-namespace}-avatar;
 
   // placeholder background
   &--placeholder {
-    background-color: var(--sapContent_ImagePlaceholderBackground);
+    background-color: var(--fdAvatar_Placeholder_BackgroundColor);
```

Comment:
This one can just be `var(--sapContent_ImagePlaceholderBackground);` and delete `--fdAvatar_Placeholder_BackgroundColor`